protectli/vp4670/default.nix

```diff
@@ -0,0 +1,9 @@
+{
+  imports = [
+    ../../common/cpu/intel/comet-lake
```

Comment:
which way of imports is preferred here:
a) `../../common/cpu/intel/comet-lake`
b) `inputs.self.nixosModules.common-cpu-intel-comet-lake`
?

Reply:
`a)`